src/main/resources/templates/index.html

```diff
@@ -8,6 +8,59 @@
     <title>Inventory Manager</title>
 
 
+    <!--Dan's stuff-->
+    <p th:text="${item}" />
+    <form action="#" th:action="@{/saveItem}" th:object="${item}" method="GET">
+        <label>Item name:</label>
+        <input type="text" name="name" th:field="*{name}"/><br><br>
+        <label>Item Description:</label>
+        <input type="text" name="description" th:field="*{description}"/><br><br>
+        <label>Location:</label>
+        <input type="text" name="location" th:field="*{location}"/><br><br>
+        <label>Location:</label>
+        <input type="text" name="assigned_to" th:field="*{assignedTo}"/><br><br>
+        <label>Location:</label>
+        <input type="text" name="status_id" th:field="*{statusId}"/><br><br>
+        <label>Item ID</label>
+        <input type="text" name="id" th:field="*{id}"/><br><br>
+        <button name="submit" type="submit" value="0">Submit</button>
+    </form>
+
+
+    <!--End of Dan's stuff-->
+
+    <!--Chris's Stuff-->
+    <br>
+
+    <table style="border: 1px solid; padding: 10px; text-align: center; margin-left: 20px">
```

Comment:
```suggestion
    <table class="table table-striped table-bordered" style="margin-left: 20px">
```
These classes should be able to replace all of the styles on the rows and columns